File: quant1x/data/schema/company.py
# -*- coding: utf-8 -*-
# Copyright (c) Quant1X <wangfengxy@sina.cn>.
# Licensed under the MIT License.

from dataclasses import dataclass

@dataclass
class CompanyInfoChunk:
    title: str = ""
    """标题"""
    filename: str = ""
    """文件名"""
    offset: int = 0
    """偏移量"""
    size: int = 0
    """大小"""

    

File: quant1x/data/schema/company.rs
// Copyright (c) Quant1X <wangfengxy@sina.cn>.
// Licensed under the MIT License.

/// 公司信息块
#[derive(Debug, Clone, Default)]
pub struct CompanyInfoChunk {
    /// 标题
    pub title: String,
    /// 文件名
    pub filename: String,
    /// 偏移量
    pub offset: i64,
    /// 大小
    pub size: i64,
}
